src/algebra/group_ring_action.lean

```diff
@@ -2,14 +2,29 @@
 Copyright (c) 2020 Kenny Lau. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau
-
-Group action on rings.
 -/
 
 import group_theory.group_action
 import data.equiv.ring
 import data.polynomial.monic
 
+/-!
+# Group action on rings
+
+This file defines the typeclass of monoid acting on semirings `mul_semiring_action M R`,
+and the corresponding typeclass of invariant subrings.
```

Comment:
Please mention that `algebra` does not satisfy axioms of `mul_semiring_action`.

Reply:
done